README.md

```diff
@@ -130,8 +130,8 @@ with the command `pca-wizard-templates  --targets`.
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
 | name | Page name in the format of `{assessment identifier}-{integer}-{descriptor}` (e.g. "RV0000-1-AutoForward"). | string | | yes |
-| capture_credentials | Capture all non-password input with GoPhish. Supports forwarding after submit action. | boolean | | yes |
-| capture_passwords | Capture password input by the user, but note that captured input is **stored in plain text as of GoPhish version 0.9.0.** | boolean | `False` | no |
+| capture_credentials | Capture all non-password input with Gophish. Supports forwarding after submit action. | boolean | | yes |
+| capture_passwords | Capture password input by the user, but note that captured input is **stored in plain text as of Gophish version 0.9.0.** | boolean | `False` | no |
```

Comment:
This is outside the scope of this PR, but someone should confirm if captured passwords are still stored in plain text.  If so, this comment should be updated from `0.9.0.` to whatever the current Gophish version is.

Reply:
@ameliav - Can you either take care of this or create an issue about this?

Reply:
Sure, I will create an issue linking to your comment.